docs/guide/static-deploy.md

```diff
@@ -87,7 +87,7 @@ Now the `preview` method will launch the server at http://localhost:8080.
    # git push -f git@github.com:<USERNAME>/<USERNAME>.github.io.git main
 
    # if you are deploying to https://<USERNAME>.github.io/<REPO>
-   # git push -f git@github.com:<USERNAME>/<REPO>.git master:gh-pages
+   # git push -f git@github.com:<USERNAME>/<REPO>.git main:gh-pages
```

Comment:
Could be that the default behavior of git is still to initiate master instead of main

Reply:
This fix is for the same reason as the commit on [line 87](https://github.com/vitejs/vite/pull/5595/files).

I also think that `master` can be the default branch, but I thought it would be appropriate to use only one of `main` and `master`.

If it is intended, I'll remove this commit. (Sorry for my English)

Reply:
I also think main is better here